docs/FAQ.md

```diff
@@ -0,0 +1,74 @@
+# Frequently Asked Questions (FAQs)
+
+## Users
+**Q : What is fAIr?**
+
+A : fAIr is an open-source toolkit developed by the Humanitarian OpenStreetMap Team (HOT) that enables the integration of artificial intelligence (AI) into humanitarian mapping workflows. It provides AI models and tools to automate mapping tasks, improving efficiency and accuracy.
+
+<hr>
+
+**Q : How can I use fAIr in my mapping projects?**
+
+A : To use fAIr, you can follow the installation instructions provided in the documentation. Once installed, you can utilize the pre-trained AI models for tasks such as road network extraction, building footprint detection, or land use classification. The documentation offers examples and guidelines to help you get started.
+
+<hr>
+
+**Q : What types of data can I use with fAIr?**
+
+A : fAIr primarily works with satellite imagery and aerial photographs. You can use various types of imagery sources, including public satellite imagery datasets, to perform mapping tasks using the AI models provided. The documentation provides information on the compatible data formats and recommended data sources.
```

Comment:
Various source of imagery which compiles with opensource imagery license

Reply:
Done